MKOB/kobwindow.py

```diff
@@ -88,14 +88,14 @@ def __init__(self, root, MKOB_VERSION_TEXT):
 
         # reader
         self.txtReader = tkst.ScrolledText(frm1, width=40, height=15, bd=2,
-                wrap='word', font=('Helvetica', -14))
+                wrap='word', font=('Arial', -14))
```

Comment:
Any reason you switched to 'Arial'? Though it is pretty well accepted now, it is (actually) a Windows font. 'Helvetica' is a generic font.

Reply:
Believe it or not, Helvetica is not installed on my Windows 10 laptop.

![Helvetica](https://user-images.githubusercontent.com/63867213/87006638-deb59d00-c175-11ea-91a1-e1bcf4049795.png)

I'm open to suggestions for a suitable font that comes preinstalled on all three platforms. The font needs to support the Narrow No-Break Space character at least, and ideally Thin Space and Hair Space as well.

Reply:
Since I have all three platforms, I'll look at fonts.

In our working this afternoon, I took your changes, so we can go with that for now. I think Mac maps 'Arial' to a similar (installed) font. Linix might do the same (haven't tested yet), but I don't like using 'Arial' in the code.

I'm good for now. I don't want to hold this up on a font issue.